### ISSUE TITLE
Feature: Auto generate actions

### ISSUE THREAD

**Issue by nmaludy** [StackStorm-Exchange/stackstorm-azure#6]
In Slack andy111 pointed out that Azure now has Swagger/OpenAPI spces for their REST API (https://github.com/Azure/azure-rest-api-specs).

Might be a good idea (at some point) to use this to auto-generate actions for this pack.

**Comment by LindsayHill** [comment]
Agreed. Assuming their growth continues, it will be only way to keep up velocity
